base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
One suggestion that I forgot to mention here was `pipe` and `PipeEnd`. I also considered `Piping` (as in to get data out of your pipeline, just add some `Piping`).